docs/report_queries/missing_label.md

```diff
@@ -1,10 +1,10 @@
-# Missing Label
-
-**Problem:** An entity does not have a label. This may cause confusion or misuse. Excludes deprecated entities.
-
-**OBO Foundry Principle:** [12 - Naming Conventions](http://www.obofoundry.org/principles/fp-012-naming-conventions.html)
-
-**Solution:** Add a label.
+# # Missing Label
```

Comment:
Introduction of redundant `#` for the MD file!

Reply:
Yes. Better to use the https://github.com/ontodev/robot/blob/master/docs/Makefile to generate the doc from the query file.

Reply:
Hm, actually deleting `docs/report_queries/missing_label.md` and (naively) running `make` inside docs does generate these # in the MD file ... if I didn't miss any config, the only thing that jumps into my mind now is that some sed regex might behave differently on various platforms (I am using M1 Mac) ... but not sure, will take a look later.

Reply:
I have just manually removed the #s at present.